package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "package-build-order",
-  "version": "0.2.3",
+  "version": "0.2.5",
```

Comment:
Could you please revert changes to this file? I can publish a new minor version after this PR goes in.

Reply:
(this isn't blocking a review, which I will do today)

Reply:
I reverted the changes to these files. Thanks for the feedback.

Reply:
Ok I removed package.json and package-lock.json changes

Reply:
Thanks!